config/locales/fr.yml

```diff
@@ -1119,6 +1120,34 @@ fr:
       peer_visible: "Rendre visible à leurs auteurs pairs"
       visibility_error: "Sélectionnez au moins une case dans la section de visibilité"
 
+    checkbox_criteria:
+      title: "Critères d' Checkbox"
+      move_up: "Monter"
```

Comment:
I was thinking the same thing as well, there's probably a chunk of them that could go under the criteria. If I recall, there was only 6 that differed between them.